pom.xml

```diff
@@ -3,9 +3,9 @@
   <groupId>org.getodk</groupId>
   <artifactId>javarosa</artifactId>
   <!-- Be sure to update version in build.gradle to match -->
-  <!-- snapshot release = x.x.x-SNAPSHOT -->
+  <!-- snapshot release = x.x.x-SNAPSHOT-${revision} -->
   <!-- production release = x.x.x -->
-  <version>5.0.0-SNAPSHOT</version>
+  <version>5.0.0-SNAPSHOT-${revision}</version>
```

Comment:
There's magic properties that allow this to work: https://maven.apache.org/maven-ci-friendly.html